src/main/contraband-scala/sbt/errorssummary/ReporterConfig.scala

```diff
@@ -99,47 +60,8 @@ final class ReporterConfig private (val colors: Boolean,
   }
 }
 object ReporterConfig {
-
-  def apply(): ReporterConfig =
-    new ReporterConfig(true,
-                       true,
-                       false,
-                       scala.Console.RED,
-                       scala.Console.YELLOW,
-                       scala.Console.CYAN,
-                       scala.Console.WHITE,
-                       scala.Console.UNDERLINED,
-                       scala.Console.BLUE)
-  def apply(colors: Boolean,
-            shortenPaths: Boolean,
-            columnNumbers: Boolean): ReporterConfig =
-    new ReporterConfig(
-      colors,
-      shortenPaths,
-      columnNumbers,
-      scala.Console.RED,
-      scala.Console.YELLOW,
-      scala.Console.CYAN,
-      scala.Console.WHITE,
-      scala.Console.UNDERLINED,
-      scala.Console.BLUE
-    )
-  def apply(colors: Boolean,
-            shortenPaths: Boolean,
-            columnNumbers: Boolean,
-            errorColor: String,
-            warningColor: String,
-            infoColor: String,
-            debugColor: String,
-            sourcePathColor: String,
-            errorIdColor: String): ReporterConfig =
-    new ReporterConfig(colors,
-                       shortenPaths,
-                       columnNumbers,
-                       errorColor,
-                       warningColor,
-                       infoColor,
-                       debugColor,
-                       sourcePathColor,
-                       errorIdColor)
+  
+  def apply(): ReporterConfig = new ReporterConfig(true, true, false, scala.Console.RED, scala.Console.YELLOW, scala.Console.CYAN, scala.Console.WHITE, scala.Console.UNDERLINED, scala.Console.BLUE)
+  def apply(colors: Boolean, shortenPaths: Boolean, columnNumbers: Boolean): ReporterConfig = new ReporterConfig(colors, shortenPaths, columnNumbers, scala.Console.RED, scala.Console.YELLOW, scala.Console.CYAN, scala.Console.WHITE, scala.Console.UNDERLINED, scala.Console.BLUE)
+  def apply(colors: Boolean, shortenPaths: Boolean, columnNumbers: Boolean, errorColor: String, warningColor: String, infoColor: String, debugColor: String, sourcePathColor: String, errorIdColor: String): ReporterConfig = new ReporterConfig(colors, shortenPaths, columnNumbers, errorColor, warningColor, infoColor, debugColor, sourcePathColor, errorIdColor)
 }
```

Comment:
Yup, it's the default generated code without IDE or Scalafmt formatting. I have IDEA set up with a plugin to run Scalafmt only on files that I edit. I'll reformat to match what you have.